Firmware/ultralcd.cpp

```diff
@@ -8947,6 +8947,35 @@ void lcd_experimental_toggle()
     eeprom_update_byte((uint8_t *)EEPROM_EXPERIMENTAL_VISIBILITY, oldVal);
 }
 
+#ifdef TMC2130
+void UserECool_toggle(){
```

Comment:
User? Did you mean Use?

Reply:
nope, User - _like ordinary non-farm user-selected/activated_